### DISCUSSION TITLE
Add custom logo for network programtically.

### DISCUSSION THREAD

**Discussion by tomvictor303** [WalletConnect/walletconnect-monorepo#4787] [Developer Support] [answered]
Dear everyone,

I am developing dapp with web3modal.

However, some networks does not have logo.

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/123891486/cf672d28-00bf-40dd-860b-6f6804abaef9)

Can I add logo for networks manually ?
(Especially Grove network)

I am using web3modal with JavaScript.
Best regards.

**Comment by glitch-txs** [comment, marked as answer]
Yesss https://docs.walletconnect.com/appkit/javascript/core/options#chainimages

**Reply by tomvictor303** [reply]
Thank you very much.
